mobileapps/serializers.py

```diff
@@ -11,11 +11,19 @@ class Meta:
 class MobileAppSerializer(serializers.ModelSerializer):
     provider_key = serializers.CharField(source="provider_key_decrypted", required=False)
     provider_secret = serializers.CharField(source="provider_secret_decrypted", required=False)
+    operating_system = serializers.ChoiceField(source="operating_system_choice_text", choices=OS_CHOICES, required=False)
```

Comment:
I don't think we need to set `source` property ChoiceField should handler it
http://www.django-rest-framework.org/api-guide/fields/#choicefield